docs/eventing/broker/kafka-broker.md

```diff
@@ -105,6 +105,9 @@ The above `ConfigMap` is installed in the cluster. You can edit
 the configuration or create a new one with the same values
 depending on your needs.
 
+_Note: `default.topic.replication.factor` must be less than or equal to the number of
+brokers of your Kafka cluster._
```

Comment:
```suggestion
**NOTE:** The `default.topic.replication.factor` value must be less than or equal to the number of Kafka broker instances in your cluster. For example, if you only have one Kafka broker, the replication factor should not be more than `1`.
```

Reply:
Suggestion just for formatting and to make this a little easier to understand, up to you if you want to use it :)

Reply:
Great suggestion! Thanks for making my docs understandable :smile:

Reply:
No problem! They were already understandable but my tech writer instincts die hard :joy: